src/jobs/BenchmarkJob.jl

```diff
@@ -360,8 +375,14 @@ function execute_benchmarks!(job::BenchmarkJob, whichbuild::Symbol)
     # make jlscript executable
     run(`chmod +x $(jlscriptpath)`)
     # clean up old cpusets, if they exist
-    try run(`sudo cset set -d /user/child`) end
-    try run(`sudo cset shield --reset`) end
+    try
+        run(`sudo cset set -d /user/child`)
+    catch
+    end
+    try
+        run(`sudo cset shield --reset`)
+    catch
```

Comment:
Maybe at least print warnings here? Is it really harmless if these commands throw errors?

Reply:
That'd be a question for Jarrett, since the `try` is already there. But emitting a warning seems okay to me.

Reply:
I'm inclined to just leave it for now, tbh.